man/sd_bus_call.xml

```diff
@@ -89,6 +89,10 @@
     input parameter; it can be used to propagate errors from the callback handler, it will not receive any
     error that was received as method reply.)</para>
 
+    <para>The message m passed to the callback is only borrowed, that is, the callback
```

Comment:
This needs markup: `<parameter>m</parameter>`, etc. Please copy over the original xml, not the rendered text.

Reply:
Fixed.